inst/tutorials/chapter4/standard_errors_standalone.Rmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "Standard Errors"
+runtime: shiny_prerendered
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(plotly)
+```
+
+## Sampling App
+
+This app simulates the drawing of a random sample of 20 observation from a population, with the underlying data generating process (DGP):
+
+$$Y = 5 + 2X + \nu, \quad \text{with:}$$
+$$X \sim \mathcal{N}(5, 2^2)\quad \text{and} \quad \nu \sim \mathcal{N}(0, 5^2)$$ 
+
+(so $\nu$ is the sampling variability, or noise)
+
+
+```{r, echo=FALSE}
+shinyUI(fluidPage(
+  actionButton("new", "New Draw!"),
+  plotlyOutput("sample"),
```

Comment:
in terms of layout. it would be best if the regression panel would take up only have the space. half page regression, other half both histrograms, so one can see it clearly next to each other. I am not sure how hard that is with plotly. use base R if not? there is no huge benefit of plotly here i think.

Reply:
will fix it, should be ok – it's a shiny `ui` thing

will push in #36 if that works with you?

Reply:
ok sure!